app/spec/lib/bk/compat/__snapshots__/examples/github-actions/go-test.yml.snap

```diff
@@ -8,7 +8,7 @@ steps:
   - go test -v ./...
   plugins:
   - docker#v5.7.0:
-      image: golang:latest
+      image: golang:1.20
```

Comment:
why do we need to stick to that image version?

Reply:
I forgot to add the bugfix to the description of the PR, sorry!